lib/profile-onboarding.dart

```diff
@@ -0,0 +1,207 @@
+import 'package:flutter/material.dart';
+import 'package:date_time_picker/date_time_picker.dart';
+import 'package:material_segmented_control/material_segmented_control.dart';
+import 'package:project_eureka_flutter/blank_page.dart';
+
+class ProfileOnboarding extends StatefulWidget {
+  @override
+  _ProfileOnboardingState createState() => _ProfileOnboardingState();
+}
+
+class _ProfileOnboardingState extends State<ProfileOnboarding> {
+  bool _profilePage = false;
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+
+  int _role = 0;
+  String _firstName;
+  String _lastName;
+  String _city;
+  String _birthDate;
+
+  TextStyle appBarTextStyle = TextStyle(
+    color: Colors.black,
+    fontWeight: FontWeight.bold,
+  );
+
+  Map<int, Widget> _children = {
+    0: Container(
+      padding: EdgeInsets.fromLTRB(15.0, 0.0, 15.0, 0.0),
+      child: Text(
+        'None',
+        style: TextStyle(fontSize: 20.0),
+      ),
+    ),
+    1: Container(
+      padding: EdgeInsets.fromLTRB(15.0, 0.0, 15.0, 0.0),
+      child: Text(
+        'Student',
+        style: TextStyle(fontSize: 20.0),
+      ),
+    ),
+    2: Container(
+      padding: EdgeInsets.fromLTRB(15.0, 0.0, 15.0, 0.0),
+      child: Text(
+        'Teacher',
+        style: TextStyle(fontSize: 20.0),
+      ),
+    ),
+  };
+
+  Column segmentedControl() {
+    return Column(
+      children: [
+        Container(
+          padding: EdgeInsets.fromLTRB(0.0, 0.0, 0.0, 10.0),
+          child: Text('Do you have a preference?'),
```

Comment:
I would change this question a little bit. Is it possible to make a question that would explain the user what this means a how it affects the functionality?